.github/workflows/test-runner-on-perf.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           roleId: ${{ secrets.PERF_ROLE_ID }}
           secretId: ${{ secrets.PERF_SECRET_ID }}
           secrets: |
+            secret/dsde/datarepo/perf/helm-azure applicationsecret | AZURE_CREDENTIALS_SECRET ;
```

Comment:
I also need to set the `AZURE_CREDENTIALS_APPLICATIONID` here (couldn't find it in vault so I'll add that)